internals/getSecurityHeaders.js

```diff
@@ -45,10 +45,9 @@ const getCsp = ({ isExtension, isDev }) =>
       'self';
     connect-src
       'self'
-      https://grpc.oasis.dev
-      https://testnet.grpc.oasis.dev
+      https://grpc.oasis.io
+      https://testnet.grpc.oasis.io
```

Comment:
just to get rid of this value can we bump test-inputs too?

Reply:
no. If we ever modify our migration code to also use old extension's NETWORK_CONFIG, then user's data will contain grpc.oasis.dev. And if we don't modify it to .io it will get blocked by CSP and tests might catch that mistake